benchmark/Microsoft.IdentityModel.Benchmarks/ValidateTokenAsyncTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.TestUtils;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Microsoft.IdentityModel.Benchmarks
+{
+    [Config(typeof(AntiVirusFriendlyConfig))]
+    public class ValidateTokenAsyncTests
+    {
+        JsonWebTokenHandler jsonWebTokenHandler;
+        JwtSecurityTokenHandler jwtSecurityTokenHandler;
+        SecurityTokenDescriptor tokenDescriptor;
+        string jsonWebToken;
+        TokenValidationParameters validationParameters;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(Default.PayloadClaims),
+                SigningCredentials = KeyingMaterial.JsonWebKeyRsa256SigningCredentials,
+            };
+            jsonWebToken = jsonWebTokenHandler.CreateToken(tokenDescriptor);
+            validationParameters = new TokenValidationParameters()
+            {
+                ValidAudience = "http://Default.Audience.com",
+                ValidateLifetime = true,
+                ValidIssuer = "http://Default.Issuer.com",
+                IssuerSigningKey = KeyingMaterial.JsonWebKeyRsa256SigningCredentials.Key,
+            };
+        }
+
+        [GlobalSetup(Targets = new[] { nameof(JsonWebTokenHandler) })]
+        public void JsonWebTokenSetup()
+        {
+            jsonWebTokenHandler = new JsonWebTokenHandler();
+        }
+
+        [Benchmark]
+        public async Task<TokenValidationResult> JsonWebTokenHandler() => await jsonWebTokenHandler.ValidateTokenAsync(jsonWebToken, validationParameters).ConfigureAwait(false);
```

Comment:
@westin-m Is there a way to a descriptive name in the report.
When you are running this, you know its validating a token.
When several reports are in front of us, how do we know what action we are doing?
I see several such runs, reading a token, creating a token, validating a token, getting claims, getting properties, etc.

Reply:
The name will always be the method name, but there is a way to include more, even custom, columns. https://benchmarkdotnet.org/articles/configs/columns.html

Reply:
change the method name to something more specific for example: `ValidateAsync`